sdk/src/androidTest/java/org/coralibre/android/sdk/internal/DeviceInfoTest.java

```diff
@@ -0,0 +1,71 @@
+package org.coralibre.android.sdk.internal;
+
+import org.coralibre.android.sdk.internal.device_info.ConfidenceLevel;
+import org.coralibre.android.sdk.internal.device_info.DeviceInfo;
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+
+public class DeviceInfoTest {
+    @Test
+    public void testCompareForDifferentManufacturers() {
+        DeviceInfo device1 = new DeviceInfo("asus", "P024_1", "P024", 1, 1, ConfidenceLevel.LOW);
+        DeviceInfo device2 = new DeviceInfo("blu", "C070", "C4 2019", 1, 1, ConfidenceLevel.LOW);
+        assertTrue("device 1 earlier that device 2", 0 > device1.compareTo(device2));
+        assertTrue("device 1 later that device 2", 0 < device2.compareTo(device1));
+        assertTrue("device 1 equals to device 2", 0 == device1.compareTo(device1));
```

Comment:
This should be `"device 1 equals to itself"`, also below

Reply:
done